ui/app/styles/core/buttons.scss

```diff
@@ -332,3 +331,54 @@ a.button.disabled {
     cursor: not-allowed;
   }
 }
+
+// Existing class on <Hds::Copy::Button> component, modifying to match existing UI Structure buttons
+.hds-copy-button {
+  font-weight: $font-weight-semibold;
+  box-shadow: $box-shadow-low;
+  border-radius: $radius;
```

Comment:
When we use HDS buttons these three styles will be removed, the classes below will remain so please give any feedback on structure/naming!

These adjust the svg and background of the default hds copy button
<img width="130" alt="Screenshot 2023-09-14 at 1 30 23 PM" src="https://github.com/hashicorp/vault/assets/68122737/44697cbd-993b-4c6d-b597-92fc293848f7">